### DISCUSSION TITLE
Raspberry Pi 4B controlling a Crazyflie 2.1 platform

### DISCUSSION THREAD

**Discussion by Vrinda73** [bitcraze/discussions#678] [Q&A] [answered]
Hello !
I am planning to control a crazyflie from a raspberry pi as my portable ground station. I have the following questions regarding the project,
1. Can I directly connect the crazyradio to Pi, do I need USB drivers ?
2. Pi specs- SD card with 2GB RAM, Ubuntu 18.04, ROS Melodic
3. Can I install and run the CF client on Pi ? Will it be too heavy for Pi ? If yes, can I at least send commands to the Crazyflie via radio (through a python script on Pi) ?
Thank you very much for your time on this ! Any help is greatly appreciated.

**Comment by ataffanel** [comment, marked as answer]
Hi!
I will answer in order
 1. Yes, no problem, the procedure is the same as for Linux on PC you will only need the correct udev rules to make the radio available to your user.
 2. 2G of Ram should work fine. I have a little doubt about the Ubuntu version though. We strive to support the latest Ubuntu LTS and the one before that. Today it means that the client and the lib targets Ubuntu 20.04 and 22.04. With 18.04 you might have problem due to the too old version of Python.
 3. The client and lib will run on a pi (depending on the version of python, see 2.). The graphics part of the client can be a bit heavy, but for testing it should work fine. Scripts will work without problems.

**Reply by Vrinda73** [reply]
Thank you very much Arnaud! Your answer was very helpful.
